core/modules/ccontrol/UserQueryFactory.cc

```diff
@@ -194,3 +240,41 @@ UserQueryFactory::Impl::Impl(StringMap const& m) {
 }
 
 }}} // lsst::qserv::ccontrol
+
+
+namespace {
+
+// regex for DROP TABLE [dbname.]table; both table and db names can be in quotes;
```

Comment:
It should be either `DROP TABLE table` or `DROP TABLE db.table`, square brackets mean optional stuff, and dot separates db from table if there is no db then dot should not be there.